package.json

```diff
@@ -11,8 +11,8 @@
     "test-peruse": "cross-env ELECTRON_RUN_AS_NODE=true NODE_ENV=test IS_UNPACKED=true BABEL_DISABLE_CACHE=1 node_modules/.bin/electron node_modules/jest/bin/jest.js __tests__ --notify",
     "test-watch": "yarn test peruse --watch",
     "test-exts": "cross-env ELECTRON_RUN_AS_NODE=true NODE_ENV=test IS_UNPACKED=true BABEL_DISABLE_CACHE=1 node_modules/.bin/electron node_modules/jest/bin/jest.js extensions --testPathIgnorePatterns=e2e --notify",
-    "test-e2e": "cross-env NODE_ENV=test SPECTRON_TEST=true IS_UNPACKED=true BABEL_DISABLE_CACHE=1 node runE2ETests.js e2e --runInBand",
-    "test-exts-e2e": "cross-env NODE_ENV=test SPECTRON_TEST=true IS_UNPACKED=true BABEL_DISABLE_CACHE=1 node runE2ETests.js exts-e2e --runInBand",
+    "test-e2e": "cross-env NODE_ENV=test SPECTRON_TEST=true IS_PACKED=true BABEL_DISABLE_CACHE=1 node runE2ETests.js e2e --runInBand",
```

Comment:
These e2e tests are running yarn and are not testing anything packaged, so this should not bee needed.